components/post_view/post_message_view/post_message_view.tsx

```diff
@@ -27,6 +27,7 @@ type Props = {
     theme: Theme; /* Logged in user's theme */
     pluginPostTypes?: any; /* Post type components from plugins */
     currentRelativeTeamUrl: string;
+    hasMention?: boolean;
```

Comment:
`hasMention` doesn't appear to be used by this component, so I think we can remove it here and remove the property passed in `PostBody`